Formula/coturn.rb

```diff
@@ -24,9 +25,11 @@ class Coturn < Formula
   depends_on "hiredis"
   depends_on "libevent"
   depends_on "libpq"
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
 
   def install
+    ENV["SSL_CFLAGS"] = "-I#{Formula["openssl@3"].opt_include}"
+    ENV["SSL_LIBS"] = "-L#{Formula["openssl@3"].opt_lib} -lssl -lcrypto"
```

Comment:
Kinda weird that these are needed now and not previously.

Reply:
I found that it was still picking up my `openssl@1.1` when both 1.1 and 3 are present. So it's able to find an OpenSSL installation, just the wrong one.